__tests__/app.test.js

```diff
@@ -11,4 +13,61 @@ describe('backend-anyapi routes', () => {
   afterAll(() => {
     pool.end();
   });
+
+  it('should create an order', async () => {
+    const res = await request(app)
+      .post('/api/v1/onepiece')
+      .send({ name: 'Monkey D. Luffy', crew: 'Straw Hats' });
+
+    expect(res.body).toEqual({
+      id: expect.any(String),
+      name: 'Monkey D. Luffy',
+      crew: 'Straw Hats',
+    });
+  });
+
+  it('Should be able to list characters', async () => {
+    await insert({ name: 'Monkey D. Luffy', crew: 'Straw Hats' });
+
+    const res = await request(app).get(`/api/v1/onepiece`);
+
+    expect(res.body).toEqual([
+      {
+        id: expect.any(String),
+        name: 'Monkey D. Luffy',
+        crew: 'Straw Hats',
+      },
+    ]);
+  });
+
+  it('Should be able to get character by id', async () => {
+    const character = await insert({
+      id: expect.any(String),
+      name: 'Monkey D. Luffy',
+      crew: 'Straw Hats',
+    });
+    const res = await request(app).get(`/api/v1/onepiece/${character.id}`);
+    expect(res.body).toEqual(character);
+  });
+
+  it('should be able to update a character', async () => {
+    const character = await Character.insert({
+      name: 'Monkey D. Luffy',
+      crew: 'Straw Hat',
+    });
+    const res = await request(app)
+      .patch(`/api/v1/onepiece/1`)
+      .send({ name: 'Monkey D. Luffy', crew: 'Straw Hat' });
```

Comment:
Looks as if nothing is be changed from the original so there's no real way to know if this test is working